.circleci/config.yml

```diff
@@ -165,16 +153,25 @@ jobs:
               --environment-variables coverage=true,coverageFile=/sdcard/coverage.ec \
               --directories-to-pull /sdcard --timeout 20m
             fi
-          no_output_timeout: 60m
+          no_output_timeout: 25m
 
 workflows:
   version: 2
   workflow:
     jobs:
-      - check_quality
-      - test_modules
-      - test_app
+      - compile
+      - check_quality:
+          requires:
+            - compile
+      - test_modules:
+          requires:
+            - compile
+      - test_app:
+          requires:
+            - compile
       - build_instrumented:
+          requires:
```

Comment:
Does this really require compile?

Reply:
It doesn't require is, but it does end up doing the same work. My thinking was to try and do the common step first and then let everything else run in parallel. It also makes these steps faster to rerun if we need to.